addons/web/models/models.py

```diff
@@ -272,7 +315,6 @@ def search_panel_select_multi_range(self, field_name, **kwargs):
         model_domain = AND([
             kwargs.get('search_domain', []),
             kwargs.get('category_domain', []),
-            kwargs.get('filter_domain', []),
```

Comment:
keep this for now, there is another issue, will be tackled later on, in another task

Reply:
ok